src/gui/widgets/Fader.cpp

```diff
@@ -400,12 +400,12 @@ void Fader::paintDBFSLevels(QPaintEvent * ev, QPainter & painter)
 
 
 	// Draw right levels
-	float const rightSpan = ampToDbv(m_fPeakValue_R) - minDB;
+	float const rightSpan = ampToDbfs(m_fPeakValue_R) - minDB;
 	int peak_R = height * rightSpan * fullSpanReciprocal;
 	QRect const drawRectR( center, height - peak_R, width, peak_R ); // Source and target are identical
 	painter.drawPixmap( drawRectR, *m_leds, drawRectR );
 
-	float const persistentRightPeakDBFS = ampToDbv(m_persistentPeak_R);
+	float const persistentRightPeakDBFS = ampToDbfs(m_persistentPeak_R);
```

Comment:
`int persistentPeak_L = height * (1 - (persistentLeftPeakDBFS - minDB) ...`

It's overflowing. 
`unsigned int persistentPeak_L ` seem to be a fix. Not tested this a lot though.